lib/lz4.c

```diff
@@ -1115,6 +1172,9 @@ int LZ4_compress_fast_continue (LZ4_stream_t* LZ4_stream, const char* source, ch
 }
 
 
+#ifdef __GNUC__
+#pragma GCC diagnostic ignored "-Wmissing-prototypes"
```

Comment:
These missing prototypes are intentional.
These functions are supposed to be used only by test tools.

I wish we could find a way to solve that without relying too much on pragma.